src/service/task.test.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CortexApi } from "../api/CortexApi";
+import mock from 'jest-mock-extended/lib/Mock';
+import { Entity } from "@backstage/catalog-model";
+import { ExtensionApi } from "@cortexapps/backstage-plugin-extensions";
+import { CatalogApi } from "@backstage/catalog-client";
+import { syncEntities } from "./task";
+import { Logger } from "winston";
```

Comment:
Heh I guess we don't lint/format test files right now? Could we do that if it's easy to add in?